src/views/PickupLocationModal.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="end" @click="closeShipmentAddressModal()">
+        <ion-button>
+          <ion-icon :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Select pickup location") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content>
+    <ion-card v-if="shipGroup.shipmentMethodTypeId !== 'STOREPICKUP'">
+      <ion-item lines="none">
+        <ion-label>{{ $t("Showing pickup locations near") }}</ion-label>
+      </ion-item>
+      <ion-item>
+        <ion-list>
+          <ion-label>{{ shipGroup.shipTo.postalAddress.toName }}</ion-label>
+          <ion-label color="dark">{{ shipGroup.shipTo.postalAddress.address1 }} </ion-label>
+          <ion-label color="dark">{{ shipGroup.shipTo.postalAddress.city }} {{ shipGroup.shipTo.postalAddress.country }} {{ shipGroup.shipTo.postalAddress.postalCode }}</ion-label>
+        </ion-list>
+      </ion-item>
+    </ion-card>
+    <ion-radio-group v-if="nearbyStores.length" v-model="facilityId">
+      <ion-list>
+        <ion-list-header lines="full" color="light">
+          <ion-label>{{ $t("Nearby stores") }}</ion-label>
+        </ion-list-header>
+        <ion-item v-for="store of nearbyStores" :key="store.facilityId">
+          <ion-label>{{ store.facilityName }}</ion-label>
+          <!-- Showing store distance in miles -->
+          <ion-label v-if="store.distance" slot="end">{{ store.distance }} {{ $t("mi") }}</ion-label>
+          <ion-radio :value="store.facilityId" slot="end" />
+        </ion-item>
+      </ion-list>
+    </ion-radio-group>
+    <ion-item v-else lines="none" class="ion-text-center">
+      <ion-label>{{ $t("No nearby stores found") }}</ion-label>
+    </ion-item>
+    <!-- Only show select button if there are stores to select from -->
+    <div v-if="nearbyStores.length" class="ion-text-center">
+      <ion-button @click="updateShipmentStore()">{{ $t("Select pickup location") }}</ion-button>
```

Comment:
I think this can be moved up where we are checking for nearbyStores.

Reply:
The button is not rendering as expected in the ion-list above. Hence, a separate block to check.

Reply:
I think you can create a parent div with v-if condition containing ion-list and ion-button.